velox/docs/develop/scalar-functions.rst

```diff
@@ -582,6 +582,58 @@ are created and then when they are copied again to the Velox vector. Some work i
 avoid that by using writer proxies that write directly to Velox vectors. This section will
 be updated once the new writer interfaces are completed.
 
+Variadic Arguments
+^^^^^^^^^^^^^^^^^^
+
+The last argument to a simple function may be marked "Variadic". This means
+invocations of this function may include 0..N arguments of that type at the end
+of the call.  While not a true type in Velox, "Variadic" can be thought of as a
+syntactic type, and behaves somewhat similarly to Array.
+
+==========  ==============================  =============================
+Velox Type  C++ Argument Type               C++ Actual Argument Type
+==========  ==============================  =============================
+VARIADIC       arg_type<Variadic<E>>              VariadicView<VectorOptionalValueAccessor<VectorReader<E>>>>
```

Comment:
are those spaced needed for rendering (the ones before VariadicView<VectorOptionalValueAccessor<Ve

Reply:
No, that's my bad, let me fix it, they're supposed to align with the lines above for readability in this file (it doesn't make a difference to rendering)